R/serializer.R

```diff
@@ -12,7 +12,7 @@
 #' @describeIn register_serializer Register a serializer with a name
 #' @export
 register_serializer <- function(name, serializer, verbose = TRUE) {
```

Comment:
Let's add a bit to the docs for `@param serializer` (maybe an example, too?)

Reply:
Mention that serializer should be a function that accepts: `ser(value, req, res, private$errorHandler)`

Reply:
Perhaps also point to "helpers" for implementing serializers like `serializer_content_type()`

Reply:
And `serializer_device()`